src/SFA.DAS.EmployerFinance/Startup/ConfigurationStartup.cs

```diff
@@ -14,8 +14,9 @@ public static IHostBuilder ConfigureDasAppConfiguration(this IHostBuilder builde
                 .AddJsonFile("appsettings.json", true, true)
                 .AddJsonFile($"appsettings.{c.HostingEnvironment.EnvironmentName}.json", true, true)
                 .AddEnvironmentVariables()
+                //todo: swap next 2, so can override table config?
```

Comment:
I agree, I think the reason we decided to call them in this order was to be consistent with the order when using `WebHostBuilder` e.g. In `SFA.DAS.EmployerFinance.Web.Program.cs` we have:

```c#
WebHost.CreateDefaultBuilder(args)
    .ConfigureDasAppConfiguration()
```